src/templates/ja/2019/featured_chapters.html

```diff
@@ -10,72 +10,72 @@
-  {%- set featured_chapter_stats = {"stat1":"93%","label1":"Pages with 3P","stat2":"49%","label2":"3P requests","stat3":"28%","label3":"3P bytes"} %}
+  {%- set featured_chapter_name = "サードパーティ" %}
+  {%- set featured_chapter_quote = "オープンウェブは広大で、リンク可能で、相互運用可能な設計になっています。他の誰かの複雑なライブラリを取得し、単一の <code>&lt;link&gt;</code> または <code>&lt;script&gt;</code> 要素を使用して自分のサイトで使用する能力は、開発者の生産性を高め、素晴らしい新しいウェブ体験を可能にしました。本章では、2019年のWebにおけるサードパーティコードの普及と影響、サードパーティソリューションの人気につながるWebの利用パターン、Webのパフォーマンスとプライバシーの将来に対する潜在的な影響を検証します。" %}
+  {%- set featured_chapter_stats = {"統計1":"93%","ラベル1":"サードパーティを含むページ","統計2":"49%","ラベル2":"サードパーティのリクエスト","統計3":"28%","ラベル3":"サードパーティのバイト数"} %}
 {%- elif featured_chapter == "fonts" %}
   {%- set featured_chapter_name = "Fonts" %}
   {%- set featured_chapter_quote = "Web fonts enable beautiful and functional typography on the web. Using web fonts not only empowers design, but it democratizes a subset of design, as it allows easier access to those who might not have particularly strong design skills. However, for all the good they can do, web fonts can also do great harm to your site's performance if they are not loaded properly." %}
   {%- set featured_chapter_stats = {"stat1":"74.9%","label1":"3P Font Requests that use Google Fonts","stat2":"29%","label2":"Percent of pages that include a Google Fonts stylesheet link","stat3":"718","label3":"Most font requests on a single page"} %}
 {%- elif featured_chapter == "performance" %}
-  {%- set featured_chapter_name = "Performance" %}
-  {%- set featured_chapter_quote = "Performance is a visceral part of the user experience. For many websites, an improvement to the user experience by speeding up the page load time aligns with an improvement to conversion rates. Conversely, when performance is poor, users don't convert as often and have even been observed to be rage clicking on the page in frustration." %}
-  {%- set featured_chapter_stats = {"stat1":"13%","label1":"Sites with fast FCP","stat2":"42%","label2":"Sites with slow TTFB","stat3":"40%","label3":"Sites with fast FID"} %}
+  {%- set featured_chapter_name = "パフォーマンス" %}
+  {%- set featured_chapter_quote = "パフォーマンスはユーザー体験の内臓的な部分です。多くのウェブサイトでは、ページの読み込み時間を高速化してユーザー体験を向上させることは、コンバージョン率の向上につながります。逆に、パフォーマンスが悪いと、ユーザーは頻繁にコンバージョンしないし、不満でページを激怒してクリックしてしまうことさえ観察されています。" %}
+  {%- set featured_chapter_stats = {"統計1":"13%","ラベル1":"Sites with fast FCP","統計2":"42%","ラベル2":"Sites with slow TTFB","統計3":"40%","ラベル3":"FIDが速いサイト"} %}
```

Comment:
Missed translating the first two labels.

Reply:
Thanks, I translated.